tqdm/_tqdm.py

```diff
@@ -462,7 +384,14 @@ def __new__(cls, *args, **kwargs):
         # Create the monitoring thread
         if cls.monitor_interval and (cls.monitor is None or not
                                      cls.monitor.report()):
-            cls.monitor = TMonitor(cls, cls.monitor_interval)
+            try:
+                cls.monitor = TMonitor(cls, cls.monitor_interval)
+            except Exception as e:  # pragma: nocover
+                from warnings import warn
+                warn("tqdm:disabling monitor support"
+                     " (monitor_interval = 0) due to:\n" + str(e),
+                     RuntimeWarning)
```

Comment:
@kalefranz you OK with this?

Reply:
I'd make a custom warning, so that it can be specifically disabled without disabling all `RuntimeWarning`s.

```python
class TqdmDisabledMonitorWarning(RuntimeWarning):
    """Warned when the tqdm update interval monitor is disabled due to insufficient threads."""
```

And then, in application code, I'd just import the warning, and add

```python
warnings.simplefilter('ignore', TqdmDisabledMonitorWarning)
```

The alternative is forcing applications to disable all `RuntimeWarning` which I don't think is what you want as a library author.  By inheriting from `RuntimeWarning` though, you allow users that flexibility if they want it.